tests/test_smoke.py

```diff
@@ -63,9 +63,6 @@ def __init__(self, name, legality):
     MockCard(  # Legal in 99 despite being a commander
         name="Slippery Bogle", legality=Legality.LEGAL
     ),
-    MockCard(  # Legal only because of an MTGA printing
-        name="Waterkin Shaman", legality=Legality.LEGAL
-    ),
```

Comment:
Worth replacing this with a new unit test you think? Like maybe one that verifies that Spiritual Guardian is NOT_LEGAL despite its printing in MTGA?

Reply:
hmmm yeah that's probably a good idea, can't hurt.

Something I've been thinking about though is that these are tests that work on specific cards but should ANOTHER card be reprinted in arena at a weird rarity....the tests won't catch it. It'll only catch Spiritual Guardian and the chance of that coming again is small.

Really, we should rework the testing to instead generate fake cards with specific attributes that we're trying to test for (from an arena set, common, etc) and then verify whether they get added or ignored to the final doc NOT specific cards we check by name